### DISCUSSION TITLE
Keyboard shortcuts for actions

### DISCUSSION THREAD

**Discussion by ogourment** [zidoro/pomatez#291] [Q&A]
I'm wondering how to make the best use of keyboard shortcuts in Pomatez.
I'd love to be able to mark the current action as done using a shortcut ("D" for Done or something like that). Currently, on the Timer screen, one can use the space key to pause/resume the timer. There is a mention of Esc but I couldn't find how/where to use it.
At a minimum, the website/doc should be a little more explicit and/or have video tutorials.
Thanks!

**Comment by ogourment** [comment]
Should I create a Feature Request instead, at least for the "Done" shortcut?

**Reply by sekwah41** [reply]
Sure :)
